src/server/controllers/VerificationController.ts

```diff
@@ -713,6 +851,50 @@ export default class VerificationController
     res.send(getSessionJSON(session));
   };
 
+  private addInputContractEndpoint = async (req: Request, res: Response) => {
```

Comment:
What was the purpose of this function? I couldn't find it in the docs either.

Reply:
Oh I forgot to add it in the docs, it takes as input an address and a chain of a deployed contract, and imports all of its files in the session